compose/xui.yml

```diff
@@ -3,7 +3,7 @@ version: '2.4'
 
 services:
   xui:
-    image: "hmctspublic.azurecr.io/xui/webapp:latest"
+    image: "hmctspublic.azurecr.io/xui/webapp:pr-827"
```

Comment:
latest if you want to merge

Reply:
Looks like they have just merged it so will do